CoreStubs/ArchiveControl/module.php

```diff
@@ -23,6 +24,25 @@ private function SetVariableData($VariableID, $Data)
         $this->Archive[$VariableID] = $Data;
     }
 
+    public function StubsAddAggregatedValues(int $VariableID, int $AggregationLevel, array $AggregationData)
```

Comment:
Warum hattest du dich entschieden den GetLogginStatus check der in AddLoggedValues drin ist hier zu entfernen?